loader_hub/file/pymu_pdf/README.md

```diff
@@ -0,0 +1,19 @@
+# PyMuPDF Loader
+
+This loader extracts text from a local PDF file using the `PyMuPDF` Python library. This is the fastest among all other PDF parsing options available in `loader_hub`. If `metadata` is passed as True during initialization; extracted documents will include basic metadata such as page numbers, file path and total number of pages in pdf.
```

Comment:
is it actually the "fastest"?

Reply:
as mentioned here (https://medium.com/social-impact-analytics/comparing-4-methods-for-pdf-text-extraction-in-python-fd34531034f). I have also personally used it and it seemed faster compared to PyPDF and PDFMiner.